pkg/controller/disruption/disruption.go

```diff
@@ -226,15 +226,11 @@ func (dc *DisruptionController) getPodStatefulSet(pod *v1.Pod) (*controllerAndSc
 }
 
 // getPodDeployments finds deployments for any replicasets which are being managed by deployments.
-func (dc *DisruptionController) getPodDeployment(pod *v1.Pod) (*controllerAndScale, error) {
-	controllerRef := metav1.GetControllerOf(pod)
-	if controllerRef == nil {
-		return nil, nil
-	}
+func (dc *DisruptionController) getPodDeployment(controllerRef *metav1.OwnerReference, namespace string) (*controllerAndScale, error) {
 	if controllerRef.Kind != controllerKindRS.Kind {
```

Comment:
need to also make sure `schema.ParseGroupVersion(controllerRef.APIVersion).Group == "apps" or "extensions"`